rubocop-rspec.gemspec

```diff
@@ -37,6 +37,7 @@ Gem::Specification.new do |spec|
   }
 
   spec.add_runtime_dependency 'rubocop', '>= 0.68.1'
+  spec.add_runtime_dependency 'rubocop-ast'
```

Comment:
With `rubocop` 1.0, and explicit requirement of `rubocop-ast` becomes unnecessary.